### ISSUE TITLE
Queryverse isn't currently usable.

### ISSUE THREAD

**Issue by blairn** [queryverse/Queryverse.jl#31]
Parquet.jl has changed their exports, which makes ParquetFiles.jl fail.

Since Queryverse relies on ParquetFiles, you can't just install Julia, add Queryverse, and use it, since it will fail with....

    julia> using Queryverse
    [ Info: Precompiling Queryverse [612083be-0b0f-5412-89c1-4e7c75506a58]
    ERROR: LoadError: UndefVarError: RecCursor not defined
    Stacktrace:
     [1] top-level scope at /home/ubuntu/.julia/packages/ParquetFiles/cLLFb/src/ParquetFiles.jl:26
     [2] include(::Module, ::String) at ./Base.jl:377
     [3] top-level scope at none:2
     [4] eval at ./boot.jl:331 [inlined]
     [5] eval(::Expr) at ./client.jl:449
     [6] top-level scope at ./none:3
    in expression starting at /home/ubuntu/.julia/packages/ParquetFiles/cLLFb/src/ParquetFiles.jl:26`

Ask ParquetFiles.jl to update to using RecordCursor, or... get Parquet.jl to reverse their last update, or, if you are an end user trying to make this work - lock Parquet.jl to version 0.4.0 until it is sorted.

like so.

    (@v1.4) pkg> add Parquet@0.4.0
      Resolving package versions...
      Installed Thrift ─── v0.6.2
      Installed Parquet ── v0.4.0
      Installed ProtoBuf ─ v0.8.0
       Updating `~/.julia/environments/v1.4/Project.toml`
      [626c502c] + Parquet v0.4.0
       Updating `~/.julia/environments/v1.4/Manifest.toml`
      [626c502c] ↓ Parquet v0.5.0 ⇒ v0.4.0
      [3349acd9] + ProtoBuf v0.8.0
      [8d9c9c80] ↓ Thrift v0.7.0 ⇒ v0.6.2
      [815b9798] - ThriftJuliaCompiler_jll v0.11.0+0
       Building Thrift → `~/.julia/packages/Thrift/Xjowa/deps/build.log`

    julia> using Queryverse
    [ Info: Precompiling Queryverse [612083be-0b0f-5412-89c1-4e7c75506a58]
    
    julia>

**Comment by proudindiv** [comment]
I've run into this too.  But it  brings up a much bigger concern that I have with Queryverse and the julia language.  After julia 1.0 we were promised that packages would settle down, but August 2018 was a long time ago now.  Packages constantly break and tutorials seldom work.  I've tried using Queryverse, like I used tidy in R, but there is no system to maintain stability here.  Google sometimes works, but usually it only finds old obsolete problems.

Does the instability of Queryverse mean that users should only use the underlying packages they need so that unrelated Package failures don't completely sabotage them?  Show me where this instability problem is discussed in any of the tutorials, books, or discussion for new users.  Most of their examples break all the time the same type of failures anyway.

When it's working, I'm pleased with julia and the Queryverse, but I lose weeks when underlying Packages that I'm not using break.
